.github/workflows/build-docs.yml

```diff
@@ -5,15 +5,19 @@ on:
   workflow_call:
     inputs:
         python-version:
+          description: 'used Python version'
           required: true
           type: string
         package-manager:
+          description: 'pip or poetry'
           required: true 
           type: string
         dependency-path:
+          description: 'Path to the dependency file'
```

Comment:
Use consistent capitalization: above you started with `used`, now with `Path`.

Personally, I slightly prefer `Path` (capitalize first word), but you decide. Just be consistent :)